opera-developer/PKGBUILD

```diff
@@ -39,6 +39,6 @@ package() {
 	
 	unlink ${pkgdir}/usr/bin/${pkgname}
 	cd ${pkgdir}/usr/bin
-	ln -sr ../usr/lib/opera-developer/opera-developer
+	ln -sf /usr/lib/${pkgname}/${pkgname}
 }
```

Comment:
I'll have to look into this. At first glance I don't see a problem. I'll check on it ASAP. Thanks!